CONTRIBUTING

```diff
@@ -21,7 +21,7 @@ guidelines:
     https://www.openssl.org/policies/cla.html for details.  If your
     contribution is too small to require a CLA (e.g. fixing a spelling
     mistake), place the text "CLA: trivial" on a line by itself separated by
-    an empty line from the rest of the body of the commit message. It is not
+    an empty line from the rest of the commit message. It is not
```

Comment:
nit: since this line has been shortened, it would be better to place "not sufficient" together on line 24, because the two words belong together and also the paragraph would look less ragged.

https://github.com/openssl/openssl/blob/9cc7462d8dcb14a8827f99a1498b2ef437a41b70/CONTRIBUTING#L23-L25

Reply:
Done and squashed.